### DISCUSSION TITLE
A way to remove opacity & transform properties from slides?

### DISCUSSION THREAD

**Discussion by suxscribe** [nolimits4web/swiper#5522] [Q&A]
Swiper 7.4.1. By default slides are switching by changing **style** attribures: _opacity_ and _transform_.
Is there a way to disable that behavior? I want only css classes to change on slide change. And no **style** attribute on slides at all.

Now:
`<li class="main-slider__item swiper-slide swiper-slide-visible swiper-slide-active" style="width: 881px; opacity: 1; transform: translate3d(0px, 0px, 0px); transition-duration: 0ms;">`
I need this:
`<li class="main-slider__item swiper-slide swiper-slide-visible swiper-slide-active">`

**Comment by Arhell** [comment]
Alternatively, you can override the styles in the css file + !important

**Reply by suxscribe** [reply]
!important is a way, but I use keyframe animations, so !important won't work in my case
